downstream/modules/eda/con-modifying-memory-limit.adoc

```diff
@@ -0,0 +1,14 @@
+[id="modifying-memory-limit"]
+
+=  Modifying the default memory limit for each rulebook activation
+
+[role="_abstract"]
+Memory usage is based on the number of events that {EDAcontroller} has to process. Each rulebook activation container has a 200mb memory limit. For example, with 4 CPU and 16GB of RAM, one rulebook activation container with an assigned 200mb memory limit can not handle more than 150K events per minute. If the number of parallel running rulebook activations is higher, then the maximum number of events each rulebook activation can process is reduced. If there are too many incoming events at a very high rate, the container can run out of memory trying to process the events. This will kill the container, and your rulebook activation will fail with a status code of 137. 
```

Comment:
```suggestion
Memory usage is based on the number of events that {EDAcontroller} has to process.
Each rulebook activation container has a 200MB memory limit.
For example, with 4 CPU and 16GB of RAM, one rulebook activation container with an assigned 200MB memory limit can not handle more than 150k events per minute.
If the number of parallel running rulebook activations is higher, then the maximum number of events each rulebook activation can process is reduced.
If there are too many incoming events at a very high rate, the container can run out of memory trying to process the events. This will kill the container, and your rulebook activation will fail with a status code of `137`. 
```
Megabytes used "MB"
Lowercase b means "byte"
And in general, k (for 1000) is lowercase
But maybe check with someone else just in case!